ui/ui-components/pages/model/[modelId]/source/[sourceId]/property/[propertyId]/edit.tsx

```diff
@@ -372,9 +372,7 @@ const Page: NextPageWithInferredProps<typeof getServerSideProps> = ({
                   </Form.Group>
                   <Form.Group controlId="sourceId">
                     <Form.Label>Property Source</Form.Label>
-                    <Form.Control as="select" disabled value={source.id}>
-                      <option value={source.id}>{source.name}</option>
-                    </Form.Control>
+                    <Form.Control as="input" disabled value={source.name} />
```

Comment:
I believe the intention was to remove these completely if the info was already on the page, which is the case for properties and apps.

If there are other models that don't have the info elsewhere then changing to an input seems ok.